src/components/task/uc/deleteUserData.uc.unit.test.js

```diff
@@ -0,0 +1,332 @@
+const sinon = require('sinon');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const { ObjectId } = require('mongoose').Types;
+
+// const chaiAsPromised = require('chai-as-promised');
+const { deleteUserRelatedData } = require('./deleteUserData.uc');
+const repo = require('../repo/task.repo');
+const { toString: idToString } = require('../../../helper/compare').ObjectId;
+// const testObjects = require('../../../../test/services/helpers/testObjects');
+
+const { expect } = chai;
+const mapByName = (deleteUserDataFunctions) =>
+	deleteUserDataFunctions.reduce((map, fun) => {
+		map[fun.name] = fun;
+		return map;
+	}, {});
+
+chai.use(chaiAsPromised);
+
+const getExpectedUpdateMany = (modifiedDocuments) => ({ success: true, modifiedDocuments });
+const getExpectedDeleteMany = (deletedDocuments) => ({ success: true, deletedDocuments });
+
+describe.only('in "deleteUserData.uc" the function', () => {
+	describe('deleteUserData', () => {
+		const userId = new ObjectId();
+		const replaceUserId = new ObjectId();
+
+		// true result
+		it('should return a function that return an array.', async () => {
+			const result = deleteUserRelatedData();
+			expect(result).to.be.an('array').with.lengthOf(5);
```

Comment:
We might just check a non-empty array (maybe with a specific length) and then loop over the elements to check all elements to be a function.